publish/working-with-pipelines/working-with-pipelines.md

```diff
@@ -210,7 +210,21 @@ status:
   internalRegistryHostname: image-registry.openshift-image-registry.svc:5000
 ```
 
-- Once you have the external route and the `ca.crt` for the internal image registry external route, use the steps in section, `Private image registries`.  @aadeshpa how do you get the ca.crt?  can we add a pointer here?
+- Once you have the external route setup you can find the ca.crt value for the external route as below from the cluster
```

Comment:
change to:
Once you have the external route set up, you can find the value for the external route in the `ca.crt` certificate.

Reply:
It should be something like this : Once you have the external route set up, you can find the ca certificate value for the external route hostname by below steps from the cluster.

Reply:
Thanks for the clarification. For accessibility reasons (people using screen readers) we're not supposed to use directional terms like "above" and "below". So let's go with:

Once you have the external route set up, you can find the ca certificate value for the external route hostname by following these steps from the cluster.

Reply:
sure, looks fine.

Reply:
updated this in the same branch.